### ISSUE TITLE
[FEATURE REQUEST] Ability to merge multiple artist name in to one artist

### ISSUE THREAD

**Issue by mdn522** [felhag/lastfm-stats-web#21]

**Comment by felhag** [comment]
Hey! Sounds like a nice feature indeed. I think [this](https://www.last.fm/api/show/artist.getCorrection) endpoint can be used to automatically correct the artist name. The only downside is I'll have to make a request for each artist, not sure the last.fm api likes to receive this much requests in a short amount of time..

**Comment by mdn522** [comment]
Yeah you are right about the requests. it's way too many. unless we have a caching system or maybe manual option

 I guess I can manually edit the exported JSON file and re-import it for now 

Again Thanks for the cool project

**Comment by felhag** [comment]
I'll keep this issue open for now, might be a cool feature to add in the future. I was already thinking of creating some kind of way to sanitize your data but that was mostly aimed at track names (removing "remastered", "2022 mix" and that kind of stuff from the title name). Merging similar artists would combine really well with this I imagine.

**Comment by mdn522** [comment]
> I'll keep this issue open for now, might be a cool feature to add in the future. I was already thinking of creating some kind of way to sanitize your data but that was mostly aimed at track names (removing "remastered", "2022 mix" and that kind of stuff from the title name). Merging similar artists would combine really well with this I imagine.

I think this particular project does something similar
https://github.com/alexbisaillion/split-scrobble-finder

It was hosted on heroku free tier. which(free tier) is not available at this moment though. but, local hosting works perfectly
